lang/cs/Org.Apache.REEF.Tests/Functional/IMRU/IMRUCloseTaskTest.cs

```diff
@@ -37,17 +40,11 @@ namespace Org.Apache.REEF.Tests.Functional.IMRU
     public class IMRUCloseTaskTest : IMRUBrodcastReduceTestBase
     {
         private const string CompletedTaskMessage = "CompletedTaskMessage";
-        private const string FailTaskMessage = "FailTaskMessage";
+        private const string FailEvaluatorMessage = "FailEvaluatorMessage";
```

Comment:
Originally the exception in close handler results in FailedTask. With REEF-1477, it is changed to FailedEvaluator. That was the reason I changed it from FailedTask to FailedEvaluator.

Now since we are not going to throw exception in the close handler, I would still add both but for verification purpose.